features/iframes.feature

```diff
@@ -8,6 +8,16 @@ Feature: IFrame interaction
     When I navigate to the home page
     Then I can locate the iframe by index
```

Comment:
Can we add the extra cuke here (Will need writing).

Don't need to do it for the id one as the final test duplicates that - I need to check all this stuff over later. But this definitely can get added now.

Reply:
@luke-hill Can you explain more about the extra cuke I should add? Do you mean I should be adding these steps?
```
Then I can see elements in an iframe with capybara query options
And I can see an expected bit of text
```